src/app/main.tsx

```diff
@@ -4,12 +4,27 @@ import App from './App.tsx';
 import { ThemeProvider } from '@emotion/react';
 import { theme } from '@styles/tokens.ts';
 import { ToastContainer } from 'react-toastify';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      staleTime: 1000 * 5,
+      gcTime: 1000 * 60 * 5,
+      retry: 1,
+    },
+  },
+});
```

Comment:
[참고] mutation에 대한 기본 옵션도 설정하면 좋겠네요

```suggestion
const queryClient = new QueryClient({
  defaultOptions: {
    queries: {
      staleTime: 1000 * 5,
      gcTime: 1000 * 60 * 5,
      retry: 1,
    },
    mutations: {
      throwOnError: ...
    },
  },
});
```

Reply:
다음 과제 내용이 에러 바운더리 적용이라 throwOnError는 일단 false로 해두고 다음 과제 때 적용해보겠습니다